inc/ad-servers/class-ad-layers-dfp.php

```diff
@@ -282,9 +267,11 @@ public function header_setup() {
 				do_action( 'ad_layers_dfp_custom_targeting' );
 				?>
 
-				if ( typeof AdLayersAPI === 'undefined' || ! AdLayersAPI.isDebug() ) {
-					googletag.enableServices();
-				}
+				<?php if ( apply_filters( 'ad_layers_dfp_enable_services_in_head', true, $this ) ) : ?>
```

Comment:
I know none of the other filters are documented, but there's no better opportunity than with this one to add a docblock for the filter 😄

Reply:
Good call!